### ISSUE TITLE
does wtdb2 remove adapters after sequencing in Nanopore ?

### ISSUE THREAD

**Issue by Velastegui** [ruanjue/wtdbg2#126]
Hi!
I am curious about if wtdb2 removes adapters after sequencing in Nanopore. Looking forward to hearing from you. 

Thanks!

**Comment by tseemann** [comment]
Until you get an answer, you could try Porechop: https://github.com/rrwick/Porechop

**Comment by ruanjue** [comment]
Thanks @tseemann, wtdbg2 doesn't remove adapters by itself. Please find other tools, e.g. porechop, to do it.

**Comment by Velastegui** [comment]
Thank you for your response. I already tried Porechop. I ran the command on my guppy sequences and I converted them to fasta files:

```porechop -i r18_guppy_combine.fastq -o porechop_R18_GUPPY.fastq.gz 
sed -n '1~4s/^@/>/p;2~4p' porechop_R18_GUPPY.fastq.gz > porechop_R18_conversion.fasta.gz
``` 
 and I started doing wtdb2 like this: 
```
./wtdbg2 -t 96 -i porechop_R18_conversion.fasta.gz -fo 4dbg  
./wtpoa-cns -t 16 -i 4dbg.ctg.lay.gz -fo 4dbg.ctg.slf.fa  
```
When I ran the last command I get this warning: 
```
t -- otal memory      264105192.0 kB 
-- available         198042208.0 kB 
-- 32 cores 
-- Starting program: ./wtpoa-cns -t 16 -i 4dbg.ctg.lay.gz -fo 4dbg.ctg.slf.fa 
-- pid                      3223 
-- date         Mon Jul  1 19:42:28 2019 
-- 
** WARNNING: input file is not in gzip format ** 
0 contigs 1 edges 
** PROC_STAT(TOTAL) **: real 0.323 sec, user 0.000 sec, sys 0.030 sec, maxrss 11120.0 kB, maxvsize 2213556.0 kB 
```
Therefore I cannot continue with: 

```
minimap2 -t 16 -x map-pb -a 4dbg.ctg.slf.fa porechop_R18_conversion.fasta.gz | best_sam_hits4longreads | samtools sort -@16 > 4dbg.ctg.map.srt.bam 
samtools view 4dbg.ctg.map.srt.bam | ./wtpoa-cns -t 16 -d 4dbg.ctg.slf.fa -i - -fo 4dbg.ctg.lrp.fa 
```
Does anyone can explain me what is going wrong.
Thanks in advance!
